juju/controller.py

```diff
@@ -257,13 +263,20 @@ def logout(self, force=False):
         """
         pass
 
-    def get_model(self, name):
+    async def get_model(self, uuid):
```

Comment:
It is still WIP. I had hoped to work on it during this sprint and wanted to note there was some code in flight in the issue #73 

This isn't ready to land, has no tests, etc.